config/en/mkdocs.yml

```diff
@@ -150,7 +150,8 @@ nav:
   - Wallet:
     - status-wallet/index.md
   - Your profile:
```

Comment:
Here you need to replace the heading "Your profile" with "Your profile and preferences"